tox.ini

```diff
@@ -384,6 +385,7 @@ deps =
     google-api-python-client<=2.60.0
     idna<=3.3
     protobuf<=3.20.1
+    hypothesis==6.21.6
```

Comment:
Please update the test command to ignore the hypothesis pytest plugin since the interference from the plugin can have some unwanted side effects

https://github.com/valory-xyz/open-autonomy/blob/main/tox.ini#L80

Reply:
Interesting, didn't know. If we're porting to open-autonomy anyway this might not be worth rerunning CI over in this PR